content/en/guides/training/api-reference.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Wondering if we can integrate this content within the _index.md page (for example). The experience feels a little disjoint to see this page in the left TOC, but then ask users to go to the reference docs.